basics.js

```diff
@@ -4,19 +4,22 @@
 
 var pets = ['cat', 'dog', 'wombat'];
 
+console.log(pets.each)
 ///// Question 2
 // Change my friend's hair color to "green", without modifying the line below.
 
 var friend = { name: 'Chris', age: 27, hairColor: 'blue' };
+this.hairColor = 'green'
```

Comment:
This refers to the current object, which in this case would be `window`, not the friend.